l10n_es_aeat/tests/test_l10n_es_aeat_export_config.py

```diff
@@ -163,7 +163,7 @@ def test_export_config_file(self):
                 ],
             }
         )
-        new_report = self.env["l10n.es.aeat.report"].new({"name": "Test Report"})
+        new_report = self.env["l10n.es.aeat.report"]
```

Comment:
Después de hacer el rebase ha comenzado a darme este error:
`TestL10nEsAeatExportConfig.test_export_config_file
2024-02-16T19:04:19.9567010Z Traceback (most recent call last):
2024-02-16T19:04:19.9567904Z   File "/__w/l10n-spain/l10n-spain/l10n_es_aeat/tests/test_l10n_es_aeat_export_config.py", line 166, in test_export_config_file
2024-02-16T19:04:19.9568806Z     new_report = self.env["l10n.es.aeat.report"].new({"name": "Test Report"})
2024-02-16T19:04:19.9569394Z   File "/opt/odoo/odoo/models.py", line 6396, in new
2024-02-16T19:04:19.9569844Z     record._update_cache(values, validate=False)
2024-02-16T19:04:19.9570318Z   File "/opt/odoo/odoo/models.py", line 5996, in _update_cache
2024-02-16T19:04:19.9570810Z     cache.set(self, field, value, check_dirty=False)
2024-02-16T19:04:19.9571241Z   File "/opt/odoo/odoo/api.py", line 999, in set
2024-02-16T19:04:19.9571605Z     record_id = record.id
2024-02-16T19:04:19.9572084Z AttributeError: 'l10n.es.aeat.report' object has no attribute 'id'`

Por lo visto en la última versión de odoo17 no se puede hacer .new() a un AbrstactModel ya que da error, quitando esta parte el test finaliza correctamente, así que entiendo que estará correcto.